src/com/facebook/buck/intellij/ideabuck/META-INF/plugin.xml

```diff
@@ -188,6 +188,21 @@
       <add-to-group anchor="last" group-id="MainToolBar"/>
     </group>
 
+    <group id="buck.test">
+      <action id="buck.RunSelectedTest"
+              class="com.facebook.buck.intellij.ideabuck.actions.BuckRunSelectedTestAction"
+              text="Run Test(s) with Buck."
+              description="Run Test(s) with Buck."
+              icon="/icons/buck_icon.png">
+      </action>
+      <action id="buck.DebugSelectedTest"
+              class="com.facebook.buck.intellij.ideabuck.actions.BuckRunSelectedTestAction"
```

Comment:
id let's it know that its different. I am going to refactor into two actions so we can generify making run and debug configs for any psi element, denoting a package, file, class, or method.